commit b98ef236e23c9b6894560d8617c08584af248563
Author: Yonik Seeley <yonik@apache.org>
Date:   Tue Sep 29 21:53:48 2009 +0000

    increase jetty wait time more... still getting failure on my fast box
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/solr/trunk@820101 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/src/webapp/src/org/apache/solr/client/solrj/embedded/JettySolrRunner.java b/src/webapp/src/org/apache/solr/client/solrj/embedded/JettySolrRunner.java
index 32fc872..9d8360e 100644
--- a/src/webapp/src/org/apache/solr/client/solrj/embedded/JettySolrRunner.java
+++ b/src/webapp/src/org/apache/solr/client/solrj/embedded/JettySolrRunner.java
@@ -111,7 +111,7 @@ public class JettySolrRunner
   }
 
   /** Waits until a ping query to the solr server succeeds,
-   * retrying every 200 milliseconds for a total of 20 seconds.
+   * retrying every 200 milliseconds up to 2 minutes.
    */
   public void waitForSolr(String context) throws Exception
   {
@@ -122,7 +122,7 @@ public class JettySolrRunner
 
     Exception ex = null;
     // Wait for a total of 20 seconds: 100 tries, 200 milliseconds each
-    for (int i=0; i<100; i++) {
+    for (int i=0; i<600; i++) {
       try {
         InputStream stream = url.openStream();
         stream.close();
